dotCMS/src/integration-test/java/com/dotcms/graphql/business/TypeTestCase.java

```diff
@@ -59,6 +61,7 @@ public BaseContentType getBaseType() {
         private String fieldVarName;
         private Class<? extends Field> fieldType;
         private boolean fieldRequired;
+        private String description;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field description has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5685344)